Jenkinsfile

```diff
@@ -632,6 +643,63 @@ pipeline {
       }
     }
 
+    stage('Deploy Preview') {
+      when {
+        not {
+          branch 'PR-*'
+        }
+      }
+      steps {
+        setGitHubBuildStatus('nuxeo/preview/deploy', 'Deploy nuxeo Preview', 'PENDING')
```

Comment:
You marked as resolved but kept `nuxeo/preview/deploy`, we're suggesting simply `nuxeo/preview`.

Reply:
Sorry about that. There must have been a manipulation error as I remember adjusting it.